Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = "A pure-Rust library for managing eBPF programs."
 homepage = "https://github.com/redcanaryco/oxidebpf"
 repository = "https://github.com/redcanaryco/oxidebpf"
 readme = "README.md"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
not a breaking change right?

Reply:
Because we are in `0.x.y`, anything to `x` is a breaking change or major features. Once we hit `1.x` the "major.minor.patch" applies. 

Also I am thinking of making a new PR soon to make oxidebpf set the right max entries for `BPF_MAP_TYPE_PERF_EVENT_ARRAY` maps. So we can hold on to releasing until that's out as well. I am planning on working on that today.

Reply:
Oh awesome, thanks for clarifying!

Reply:
Great, let's roll them all into one release.